src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public abstract class ColorSupplier extends Figure {

```

Comment:
The `ColorSupplier` class should not be abstract and should not extend `Figure`. It seems to be a utility class for providing random colors, so it should be a standalone class.